dfx.json

```diff
@@ -0,0 +1,14 @@
+{
+  "dfx": "0.18.0",
```

Comment:
This version is already outdated. Is there a reason not to use the latest one?

Reply:
No, good point. I copied from the VC playground. I expected that one to have the latest.